### PR TITLE
Apply docopts and long options policies

### LINKED ISSUE #848: Unify CLI options documentation
Our way of documenting CLI options varies: sometimes using `$` for variables, sometimes `<>`, sometimes using a comma for separating repeating arguments, sometimes not. This leads to recurring discussions between contributors and lack of clarity for users.

Considering [there is no agreed-upon standard](https://stackoverflow.com/questions/9725675/is-there-a-standard-format-for-command-line-shell-help-text):

- Use http://docopt.org syntax.
- Reference http://docopt.org in the CONTRIBUTING.

### PR DESCRIPTION
**Pull request by MattiSG**
Fixes #848.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -48,6 +50,35 @@ We add this additional rule:
...
+
+#### CLI
+
+##### docopt
+
+For command-line examples and documentation, we follow the [docopt usage patterns](http://docopt.org) syntax. Quick recap of the main points:
+
+- mandatory arguments are given between `<` and `>`;
+- optional elements are given between `[` and `]`;
+- mutually exclusive elements are given between `(` and `)` and separated by `|`.
```
Comment (martinratinaud) [inline CONTRIBUTING.md:65]: is there also a `default` value we should document?

like `[argument=defaultvalue]`

**Review by martinratinaud — APPROVED** [review]

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -48,6 +50,35 @@ We add this additional rule:
...
+
+```diff
+- npm start $service_id [, $service_id, ...]
++ npm start <service_id>...
```
Comment (Ndpnt) [inline CONTRIBUTING.md:69]: `service_id` is optional, shouldn't it be written like that `[service_id]` instead?

File: CONTRIBUTING.md (hunk 1 above)
Comment (MattiSG) [reply CONTRIBUTING.md:65]: This is just a recap, I don't think repeating everything from the full doc is mandatory. We can also just remove these examples if they sound limitative.

File: CONTRIBUTING.md (hunk 2 above)
Comment (MattiSG) [reply CONTRIBUTING.md:69]: I voluntarily gave an example that did not reflect accurately the current CLI but simply gave examples of how to rewrite. I will adjust it to reflect the CLI.

**Pull request merged**